src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,159 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    public MyLinkedList() {
+    }
+
+    public MyLinkedList(T element) {
+        Node<T> firstNode = new Node<>(null, element, null);
+        head = tail = firstNode;
+        size = 1;
+    }
+
+    private void checkBounds(int index) {
```

Comment:
private methods should be located after public

Reply:
done